docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
 
-## Framework-dependent deployment with third-party dependencies
+You can use the `<TargetFrameworks>` setting (note the plural) to specify more than one framework supported by your app. Separate each TFM with a semicolon. You can publish, targeting one of the frameworks listed, with the `dotnet publish -f <TFM>` command. For example, if you have `<TargetFrameworks>netcoreapp2.1;netcoreapp2.2</TargetFrameworks>`, and run `dotnet publish -f netcoreapp2.1`, an executable that targets .NET Core 2.1 is created. 
 
-Deploying a framework-dependent deployment with one or more third-party dependencies requires that those dependencies be available to your project. Two additional steps are required before you can run the `dotnet restore` ([see note](#dotnet-restore-note)) command:
+Unless otherwise set, the output directory of the [`dotnet publish`][dotnet-publish] command is `<CURRENT-FOLDER>/bin/<BUILD-CONFIGURATION>/<TFM>/publish/`. The default **BUILD-CONFIGURATION** mode is **Debug**. Based on the SDK you're using, and the target framework chosen, either a *framework-dependent deployment* or *framework-dependent executable* app is created.
 
-1. Add references to required third-party libraries to the `<ItemGroup>` section of your *csproj* file. The following `<ItemGroup>` section contains a dependency on [Json.NET](https://www.newtonsoft.com/json) as a third-party library:
+### Native dependencies
 
-      ```xml
-      <ItemGroup>
-        <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
-      </ItemGroup>
-      ```
+If your app has some sort of native dependency, it may not run on a different operating system. For example, if you (on Windows) referenced a library that used the native Win32 API, your app wouldn't run on macOS or Linux. You would need to provide platform-specific code, and compile an executable for each platform. 
 
-1. If you haven't already, download the NuGet package containing the third-party dependency. To download the package, execute the `dotnet restore` ([see note](#dotnet-restore-note)) command after adding the dependency. Because the dependency is resolved out of the local NuGet cache at publish time, it must be available on your system.
+Consider also that if a library you referenced has a native dependency, your app also may not run on every platform. However, it's possible that a NuGet package you're referencing has included platform-specific versions, handling the required native dependencies for you.
```

Comment:
referenced --> reference
handling is a dangling participle. platform-specific versions that handle...